amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -8,6 +8,7 @@ import Pagination from 'react-js-pagination';
 import SearchBar from './SearchBar';
 import SearchList from './SearchList';
 
+import BookmarkList from 'components/common/Bookmark/BookmarkList'
```

Comment:
small nit: should we enforce alphabetical order?

Reply:
Imports are loosely sorted within each group of imports. We have 3rd party imports on lines 1-6, relative imports within the same folder on lines 8-9, `/components` on lines 11-14, and `/ducks` on lines 16-27. Constants on line 32 could be grouped with other relative imports.

We don't have an explicit pattern, but we could certainly look into establishing a stronger one. For example, should we sort by import path or named imported? Should imports be grouped or alphabetize altogether? Where do styles belong? I'd be happy to revisit in a follow up task.